eclass/cuda.eclass

```diff
@@ -46,11 +46,11 @@ cuda_gccdir() {
 
 	local gcc_bindir ver args="" flag ret
 
-	# Currently we only support the gnu compiler suite
-	if [[ $(tc-getCXX) != *g++* ]]; then
```

Comment:
From your description I gather that the check is misfiring on your machine. I think it'd be better to fix than to remove it. Could you try using `tc-is-gcc` here instead?